docs/how-to/upgrade-cloud.rst

```diff
@@ -2,6 +2,103 @@
 Upgrade a cloud
 ===============
 
+Recommendations for upgrading production clouds
+-----------------------------------------------
+
+1. Be sure to upgrade in a maintenance window
+2. Start with the upgrade on control-plane applications
+3. Don't force the upgrade on non-empty hypervisors
+4. After upgrading the control-plane, choose one empty hypervisor to be a `canary node`.
+    After upgrading it, test if it's behaving as expected.
+5. If no issues are found after upgrading the `canary node`, proceed with the upgrade.
+
+Run upgrade for the whole cloud
+-------------------------------
+
+To upgrade the whole cloud in a single command.
+
+.. code:: bash
+
+    cou upgrade
+
+
+Run upgrade for the control-plane
+---------------------------------
+
+To upgrade a plan targeting the **control-plane** applications use:
+
+.. code:: bash
+
+    cou upgrade control-plane
+
+**Note:** Control-plane applications will be upgraded using the `all-in-one`_ method,
+meaning that all units will be upgraded at once.
```

Comment:
I don't think this information should belong to the "how-to" guides. I'm adding this into `explanation` in my upcoming patch.

Reply:
FYI this is part of the #323